### ISSUE TITLE
Update to 0.10.25 fails due to "Row size too large"

### ISSUE THREAD

**Issue by timdeluxe** [Froxlor/Froxlor#926]
# Bug report
# Summary

Update in panel from 0.10.24 to 0.10.25 blocked due to "Row size too large".

Detailled error: 
"42000 SQLSTATE[42000]: Syntax error or access violation: 1118 Row size too large. The maximum row size for the used table type, not counting BLOBs, is 8126. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs"

# System information
* Froxlor version: 0.10.24
* Web server: apache2
* DNS server: none
* POP/IMAP server: Dovecot
* SMTP server: postfix
* FTP server: proftpd
* OS/Version: Debian GNU/Linux 10 (buster)
* MariaDB/MySQL Version: 5.5.5-10.3.27-MariaDB-0+deb10u1

# Steps to reproduce

1. Have 0.10.24 installed
2. Install 0.10.25 update
3. Try to do the upgrade in froxlor admin panel

# Expected behavior

Update works succesfully

# Actual behavior

Update failed due to "Row size too large"

# Log files/log entries
syslog:
<pre>
Mar  8 19:35:05 server froxlor[28883]: froxlor.NOTICE: viewed admin_updates {"source":"admin","action":"30","user":"admin"} []
Mar  8 19:35:05 server froxlor[28883]: froxlor.WARNING: -------------- START LOG -------------- {"source":"admin","action":"30","user":"updater"} []
Mar  8 19:35:05 server froxlor[28883]: froxlor.WARNING: Add new description fields to mail and domain table {"source":"admin","action":"30","user":"system"} []
Mar  8 19:35:05 server froxlor[28883]: SQLSTATE[42000]: Syntax error or access violation: 1118 Row size too large. The maximum row size for the used table type, not counting BLOBs, is 8126. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs
Mar  8 19:35:05 server froxlor[28883]: --- DEBUG: #0 [internal function]: PDO->query('ALTER TABLE pan...') #1 /var/www/froxlor/lib/Froxlor/Database/Database.php(218): call_user_func_array(Array, Array) #2 /var/www/froxlor/install/updates/froxlor/0.10/update_0.10.inc.php(741): Froxlor\Database\Database::__callStatic('query', Array) #3 /var/www/froxlor/install/updatesql.php(44): include_once('/var/www/froxlo...') #4 /var/www/froxlor/admin_updates.php(62): include_once('/var/www/froxlo...') #5 {main}
</pre>

**Comment by d00p** [comment]
Currently investigating in this issue, all my test boxes went through the update flawlessly. Need to find the cause.

Quickfix:

File: __/etc/mysql/my.cnf__ add the following:

```
innodb_default_row_format = 'DYNAMIC'
innodb_strict_mode=OFF
```

**Comment by timdeluxe** [comment]
@d00p I could use the quickfix, but i have no hurry. I will leave the situation like this to help you fixing. Just tell me if you need any further information.

**Comment by florent-izypay** [comment]
> 
> 
> Currently investigating in this issue, all my test boxes went through the update flawlessly. Need to find the cause.
> 
> Quickfix:
> 
> File: **/etc/mysql/my.cnf** add the following:
> 
> ```
> innodb_default_row_format = 'DYNAMIC'
> innodb_strict_mode=OFF
> ```

Our DB server is hosting other databases that wouldn't have supported this generic server-wide param.
So, after digging a bit with the report message:

> Dear froxlor-team,
>
>the following error has been reported by a user:
>
>-------------------------------------------------------------
>42000 SQLSTATE[42000]: Syntax error or access violation: 1118 Row size too large. The maximum row size for the used table >type, not counting BLOBs, is 8126. This includes storage overhead, check the manual. You have to change some columns to >TEXT or BLOBs
>
>File: lib/Froxlor/Database/Database.php:218
>
>Trace:
>#0 [internal function]: PDO->query()
>#1 lib/Froxlor/Database/Database.php(218): call_user_func_array()
>#2 install/updates/froxlor/0.10/update_0.10.inc.php(741): Froxlor\Database\Database::__callStatic()
>#3 install/updatesql.php(44): include_once('/var/www/froxlo...')
>#4 admin_updates.php(62): include_once('/var/www/froxlo...')
>#5 {main}
>
>-------------------------------------------------------------
>>
>Froxlor-version: 0.10.25
>DB-version: 202103030
>
>End of report

The problem lies here: 
https://github.com/Froxlor/Froxlor/blob/91d4432108a6a21eb8bbab84d789c8bd5deb5826/install/updates/froxlor/0.10/update_0.10.inc.php#L744

It appeared to me that enabled `DYNAMIC` `ROW_FORMAT` ont this particular table :

```
ALTER TABLE `froxlor`.`panel_domains` ROW_FORMAT=DYNAMIC;
```

solved the problem and allows update to be executed.

**Comment by d00p** [comment]
fixed in upcoming release with c891431
